### DISCUSSION TITLE
QuickSnap_Presentation_low.mp4

### DISCUSSION THREAD

**Discussion by bisechen** [JulienHeijmans/quicksnap#5] [Ideas]
I see there is an MP4 video in your intro description, but it doesn't seem to play.

**Comment by JulienHeijmans** [comment]
Hey, thank you for reporting the issue. It's strange, it is playing for me. What web browser do you use?

**Reply by bisechen** [reply]
Hi, 
I use EDGE , but switching to Chrome doesn't play either.
Youtube playback is totally fine.

**Comment by JulienHeijmans** [comment]
Ah yes, I managed to reproduce the problem using edge. (For some reason it does play fine in Chrome for me, maybe the video gets somehow cached when I upload it?)

Anyway, i think it was because I encoded the video in h265. I re-encoded the video in h264, and now it seems to work fine.
Can you try again and see if it loads now?

Anyway, thank you for reporting the problem!

**Reply by bisechen** [reply]
This video is now ready to play.
